release/update-rsync.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```

Comment:
As we can safely assume this will be run on Linux I would raise the bar to bash; it is not a requirement for this PR.

Reply:
Well, this the server is currently Debian/Ubuntu so `/bin/sh` is an alias for Dash not Bash. Bash is a fair bit heavier in CPU and memory usage.